src/components/WorkflowsExplorer/Timeline/README.md

```diff
@@ -0,0 +1,6 @@
+## About this component
+The timeline component was extracted by @zzeekk from the open-source project Metaflow UI [https://github.com/Netflix/metaflow-ui]. 
+
```

Comment:
I think we sould mention the license here, e.g.:
"Metaflow UI is licensed under Apache-2.0 license. The corresponding license file is provided in this folder."

Reply:
Yes definitely